src/csv/builder.cr

```diff
@@ -136,13 +150,20 @@ class CSV::Builder
     end
 
     private def needs_quotes?(value)
-      value.each_byte do |byte|
-        case byte.unsafe_chr
-        when @separator, @quote_char, '\n'
-          return true
+      case @quoting
+      when .rfc?
+        if value.is_a?(String)
```

Comment:
If the value is a Char or Symbol the quoting logic is not applied.
That was a pre-existing corner case that was not handled.
Should we handled it now?

Reply:
Well, I prefer this PR just provides a new **feature** about quoting behavior.
And then, I'd like to fix the corner case soon in another PR that provides **fix the existing issue**.